src/test/java/org/openqa/selenium/htmlunit/WebDriverTestCase.java

```diff
@@ -541,19 +542,16 @@ protected WebDriver buildWebDriver() throws IOException {
             throw new RuntimeException("Unexpected BrowserVersion: " + getBrowserVersion());
         }
         if (webDriver_ == null) {
-            final WebClientOptions options = new WebClientOptions();
+            final HtmlUnitDriverOptions driverOptions = new HtmlUnitDriverOptions(getBrowserVersion());
+            
             if (isWebClientCached()) {
-                options.setHistorySizeLimit(0);
+                driverOptions.setCapability(HtmlUnitOption.optHistorySizeLimit, 0);
             }
-
-            final Integer timeout = getWebClientTimeout();
-            if (timeout != null) {
-                options.setTimeout(timeout.intValue());
+            
+            if (getWebClientTimeout() != null) {
+                driverOptions.setCapability(HtmlUnitOption.optTimeout, getWebClientTimeout());
```

Comment:
This could also be expressed as `HtmlUnitOption.TIMEOUT.getCapabilityKey()`, with the same verbosity issue.